docs/burger-discounts/index.md

```diff
@@ -407,21 +407,21 @@ using lists, which are a better fit for this problem.
 - Type inference is less effective inside functions that don't call other
   functions. In those cases, you can give the compiler more information:
   - Type annotate the function arguments
-  - Use the full name for value
+  - Use the full name for a value used inside the function
 - The `Stdlib` module is opened by default
 - Records are immutable
-- Use record copy syntax to make copies of records that have different values
-  for some fields
+- Use record copy syntax to copy a record while changing some fields on the
```

Comment:
Maybe could be like this, to avoid saying "copy" a few times:

```suggestion
- Use record spread syntax to duplicate a record while changing some fields on the
```

Reply:
Great suggestion! Rephrased in https://github.com/melange-re/melange-for-react-devs/pull/36/commits/729143b7c05e009432d746242faee156e520cdc1

I was hesitant to call it "record spread syntax" at first, because in OCaml syntax there's no spread operator, but after a little consideration I think it makes more sense to just call it that since the target audience is JS developers.